### PR TITLE
Update development-groups.html

### PR DESCRIPTION
**Pull request by wdjoyner**
Added sage-coding-theory link

### CODE REVIEW

File: src/development-groups.html [hunk 1]
```diff
@@ -142,6 +142,27 @@ <h2><a href="http://groups.google.com/group/sage-release/"
      </ul>
     </td>
 
+    <td class="rightpadding">
+     <h2><a href="http://groups.google.com/group/sage-coding-theory/"
+       >sage-finance</a></h2>
```
Comment (dimpase) [inline src/development-groups.html]: This must say `>sage-coding-theory</a></h2>`

**Comment by haraldschilly** [comment]
ok!

**Comment by haraldschilly** [comment]
Hmm ... :disappointed: I was too optimistic. I assumed you compiled the website.

screenshot from 2019-01-23 10-49-24

**Comment by dimpase** [comment]
I never tried this. Is it documented anywhere, how you can build and deploy
it on localhost?

**Comment by haraldschilly** [comment]
I just fixed it, should be fine now. 

Yes, it's documented in the [readme](https://github.com/sagemath/website#development). Essentially you have to run `make` and open up the `www` subdirectory.

**Comment by dimpase** [comment]
> Yes, it's documented in the [readme](https://github.com/sagemath/website#development)

I didn't read it carefully - it looks too much like poetry to me :-)
I'd edit out all these "ideally", etc.

What worked for me: `make` to build, starting `make server` in another shell, and keeping it running, 
then `make show` worked (after changing port to 8080, as 8181 is blocked in the setup I used (crostini/chromebook))

**Comment by dimpase** [comment]
> I don't know how to use github, obviously:-(
now you can proudly put "basic GitHub skills" in your CV ;-)

**Pull request merged**